src/pages/challenges/ChallengeTile.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Flex,
+  Box,
+  Button,
+  Text,
+  Stack,
+  Card,
+  CardBody,
+  CardFooter,
+  Image,
+  HStack,
+  Heading,
+} from '@chakra-ui/react'
+import { NavLink } from '@/components/NavLink'
+import ChallengeProps from '@/pages/challenges/challengeProps'
+
+export const ChallengeTile = ({
+  title,
+  image,
+  description,
+  level,
+  topics,
+}: ChallengeProps) => {
+  return (
+    <Card maxW='sm' h='100%' _hover={{ boxShadow: '4px 4px 7px 0px #718096' }}>
+      <CardBody>
+        <Image src={image} alt={title} borderRadius='lg' />
+        <Stack mt='6' spacing='3'>
+          <Flex alignItems='center' justifyContent='space-between'>
+            <Heading size='md'>{title}</Heading>
+            <Text
+              color={
+                level === 'Easy'
+                  ? 'green.600'
+                  : level === 'Medium'
+                  ? 'orange.300'
+                  : 'red.400'
+              }
+              fontSize='2xl'
+            >
+              {level}
+            </Text>
+          </Flex>
+
+          <Text>{description}</Text>
+        </Stack>
+      </CardBody>
+      <HStack spacing='5px' ml='24px'>
+        {topics?.map((top, id) => {
+          return (
+            <Box
+              key={id}
+              as='button'
+              height='24px'
+              lineHeight='1.2'
+              transition='all 0.2s cubic-bezier(.08,.52,.52,1)'
+              border='1px'
+              px='8px'
+              borderRadius='2px'
+              fontSize='14px'
+              fontWeight='semibold'
+              bg='#f5f6f7'
+              borderColor='#ccd0d5'
+              marginRight='4px'
+              color='#4b4f56'
+              _hover={{ bg: '#ebedf0' }}
+              _active={{
+                bg: '#dddfe2',
+                transform: 'scale(0.98)',
+                borderColor: '#bec3c9',
+              }}
+              _focus={{
+                boxShadow:
+                  '0 0 1px 2px rgba(88, 144, 255, .75), 0 1px 1px rgba(0, 0, 0, .15)',
+              }}
+            >
+              {top}
+            </Box>
+          )
+        })}
+      </HStack>
+
+      <CardFooter>
+        <NavLink href={`/${title.replace(/\s/g, '%20')}`}>
```

Comment:
Nice job using the `title` to programmatically generate the route.  Just a couple suggestions:

- To keep each challenge under the `/challenges` route, I'd suggest including that in the `href`.
- Not sure that this `.replace` is necessary, I think it may work the same without it?

<details>
<summary>Click me to see screenshots</summary>

> NOTE: the `url` at the bottom of the screenshot

#### Before this suggestion, hovering over the `VIEW` button

<img width="963" alt="Screenshot 2023-07-29 at 8 39 35 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/a392eb82-93ef-4281-9ae9-4ad59c0b2e74">

#### Before this suggestion, after clicking the `VIEW` button

<img width="968" alt="Screenshot 2023-07-29 at 8 39 48 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/00759b76-f150-439d-a558-af5c5441ee59">

#### After this suggestion, hovering over the `VIEW` button

<img width="965" alt="Screenshot 2023-07-29 at 8 41 49 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/be1de812-60f8-4214-915f-f3bf0d7a87fb">

#### After this suggestion, after clicking the `VIEW` button

<img width="962" alt="Screenshot 2023-07-29 at 8 42 03 AM" src="https://github.com/mitxpro-dev/learnjs/assets/96446064/ca520d92-40aa-4d9a-ba39-a9845dd6cff1">

</details>


```suggestion
        <NavLink href={`/challenges/${title}`}>
```